code/modules/projectiles/updated_projectiles/guns/flamer.dm

```diff
@@ -11,6 +11,8 @@
 	force = 15
 	fire_sound = "gun_flamethrower"
 	dry_fire_sound = 'sound/weapons/gun_flamethrower_empty.ogg'
+	unload_sound = 'sound/weapons/gun_flamethrower_unload.ogg'
```

Comment:
Due to the ammount of gun sounds, would it be possible to make a folder for it and remove the gun prefix?

Reply:
Can do.